docs/docfx.json

```diff
@@ -975,7 +975,7 @@
       ],
       "feedback_system": "GitHub",
       "feedback_github_repo": "OfficeDev/office-js-docs-reference",
-      "feedback_product_url": "/answers/products/m365"
+      "feedback_product_url": "/answers/tags/255/javascript-api"
```

Comment:
What's the difference between this link and the Office Development one?

Reply:
Asking because there seems to be fewer Office-specific questions with the JS tag.

Reply:
@AlexJerabek I was going by the tag description ("An Office service that supports add-ins to interact with objects in Office client applications.") and since this is API docs.

The Office Development description is:
Office: A suite of Microsoft productivity software that supports common business tasks, including word processing, email, presentations, and data management and analysis.
Development: The process of researching, productizing, and refining new or existing technologies.

I'd have no problem having them all use the same link.

Reply:
I find it funny there's a definition like that for Office...

Based on what questions the community is leaving, I'd lean towards sending our devs to Office Development.